scanner/src/main/kotlin/ScanContext.kt

```diff
@@ -60,5 +61,10 @@ data class ScanContext(
      * The packages known to be covered in the context of this scan. For package scanners, this is the list of packages
      * that have the same provenance as the reference package.
      */
-    val coveredPackages: List<Package> = emptyList()
+    val coveredPackages: List<Package> = emptyList(),
+
+    /**
+     * The [PackageSnippetChoice] of the project to scan.
+     */
+    val packageSnippetChoices: List<PackageSnippetChoice> = emptyList()
```

Comment:
Again, just `snippetChoices`? I.e. what additional value / information does "package" bring? Will there ever be non-package / project snippet choices?

Reply:
`SnippetChoices` is already taken by the container of `Choice` and `Given`.
Any suggestion ?

Maybe we can can name the property `snippetChoices` but keep the class named `PackageSnippetChoice` ?